examples/auth-generate-token/page.md

```diff
@@ -0,0 +1,151 @@
+# Generate a JWT token to authenticate clients
+
+Use JWT tokens to securely authenticate users within your application.
+
+JWT tokens enable you to grant users access to Ably's real-time messaging services. They are most commonly used to issue time-limited credentials that securely authorize users to connect to Ably channels.
+
+JWT tokens are an essential component of applications that require secure and scalable real-time communication. Before any application can use any of the Ably products whether they be Pub/Sub, Spaces, Chat, or LiveSync, they'll need to first authenticate.
+
+Authentication is implemented using [Ably core Pub/Sub](https://ably.com/docs/auth). The Pub/Sub SDK contains a set of purpose-built APIs that abstract away the complexities involved in architecting websocket features. It is built on top of Ably's core platform, and so it provides the same performance guarantees and scaling potential.
+
+## Resources
+
+// React brief (Only visible if viewing the React example)
+
+This example uses a frontend and a backend client. The frontend is a React application and the backend is an Express Node.js application. Both of these applications use the [Ably JS SDK](https://github.com/ably/ably-js).
+
+The server application calls [`auth.requestToken();`](https://ably.com/docs/auth/token#ably-token) to request a token from Ably in order to pass it to the client to be used as authentication. The client calls the server's publicly exposed endpoint `/auth-url` to request this token.
+
+Find out more about [authentication](https://ably.com/docs/auth).
+
+// End React brief
+
+// Javascript brief (Only visible if viewing the Javascript example)
+
+This example uses a frontend and a backend client. The frontend is a Typescript application and the backend is an Express Node.js application. Both of these applications use the [Ably JS SDK](https://github.com/ably/ably-js).
+
+The server application calls [`auth.requestToken();`](https://ably.com/docs/auth/token#ably-token) to request a token from Ably in order to pass it to the client to be used as authentication. The client calls the server's publicly exposed endpoint `/auth-url` to request this token.
+
+Find out more about [authentication](https://ably.com/docs/auth).
+
+// End Javascript brief
+
+## View on Github
```

Comment:
You need to add an additional step to these instructions to cover adding your API key to the server.

Reply:
I'm sure I added a comment here but I can't find it... Steps 5-10 are these steps.